src/webgl/p5.Framebuffer.js

```diff
@@ -6,1622 +6,1636 @@
+
+    this.pixels = [];
+
+    this.format = settings.format || constants.UNSIGNED_BYTE;
+    this.channels = settings.channels || (
+      target._renderer._pInst._glAttributes.alpha
+        ? constants.RGBA
+        : constants.RGB
+    );
+    this.useDepth = settings.depth === undefined ? true : settings.depth;
+    this.depthFormat = settings.depthFormat || constants.FLOAT;
+    this.textureFiltering = settings.textureFiltering || constants.LINEAR;
+    if (settings.antialias === undefined) {
+      this.antialiasSamples = target._renderer._pInst._glAttributes.antialias
+        ? 2
+        : 0;
+    } else if (typeof settings.antialias === 'number') {
+      this.antialiasSamples = settings.antialias;
+    } else {
+      this.antialiasSamples = settings.antialias ? 2 : 0;
+    }
+    this.antialias = this.antialiasSamples > 0;
+    if (this.antialias && target.webglVersion !== constants.WEBGL2) {
+      console.warn('Antialiasing is unsupported in a WebGL 1 context');
+      this.antialias = false;
+    }
+    this.density = settings.density || target.pixelDensity();
+    const gl = target._renderer.GL;
+    this.gl = gl;
+    if (settings.width && settings.height) {
+      const dimensions =
+        target._renderer._adjustDimensions(settings.width, settings.height);
+      this.width = dimensions.adjustedWidth;
+      this.height = dimensions.adjustedHeight;
+      this._autoSized = false;
+    } else {
+      if ((settings.width === undefined) !== (settings.height === undefined)) {
+        console.warn(
+          'Please supply both width and height for a framebuffer to give it a ' +
+            'size. Only one was given, so the framebuffer will match the size ' +
+            'of its canvas.'
+        );
+      }
+      this.width = target.width;
+      this.height = target.height;
+      this._autoSized = true;
     }
+    this._checkIfFormatsAvailable();
 
-    get height() {
-      return this.framebuffer.height * this.framebuffer.density;
+    if (settings.stencil && !this.useDepth) {
+      console.warn('A stencil buffer can only be used if also using depth. Since the framebuffer has no depth buffer, the stencil buffer will be ignored.');
     }
+    this.useStencil = this.useDepth &&
+      (settings.stencil === undefined ? true : settings.stencil);
 
-    rawTexture() {
-      return this.framebuffer[this.property];
+    this.framebuffer = gl.createFramebuffer();
+    if (!this.framebuffer) {
+      throw new Error('Unable to create a framebuffer');
+    }
+    if (this.antialias) {
+      this.aaFramebuffer = gl.createFramebuffer();
+      if (!this.aaFramebuffer) {
+        throw new Error('Unable to create a framebuffer for antialiasing');
+      }
     }
-  };
+
+    this._recreateTextures();
+
+    const prevCam = this.target._renderer.states.curCamera;
+    this.defaultCamera = this.createCamera();
+    this.filterCamera = this.createCamera();
+    this.target._renderer.states.curCamera = prevCam;
+
```

Comment:
@davepagurek I think the problem we are having is likely around here along with the `createCamera` method below. Notably the camera that is created for the graphics after its creation is different from what it was after the framebuffer is created, my guess is that its a reference issue, perhaps the `FramebufferCamera` class needs a `.clone()` implementation?